Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -719,6 +719,23 @@ public static class Defaults
                 Statistic = Statistic.Average,
                 Namespace = AwsNamespace.Elb,
                 TreatMissingData = TreatMissingDataConstants.NotBreaching
+            },
+            new AlarmDefinition
+            {
+                Name = "HealthyHostCountLow",
+                Metric = "HealthyHostCount",
+                Period = TimeSpan.FromMinutes(5),
+                EvaluationPeriods = 4,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 1
+                },
+                DimensionNames = new[] { "LoadBalancerName" },
+                ComparisonOperator = ComparisonOperator.LessThanOrEqualToThreshold,
+                Statistic = Statistic.Average,
```

Comment:
just wondering if minimum might be more useful? if you're averaging over 5 minutes could you get a value like 1.2 if you didn't have a box only in the last data point?